app/models/facility_group.rb

```diff
@@ -75,6 +76,10 @@ def cohort_analytics(period:, prev_periods:)
     query.call
   end
 
+  def syncable_patients
+    registered_patients.with_discarded
+  end
```

Comment:
I'm not sold on which one is better, I think getting syncable patients from the region makes sense and so does asking the patient to fetch the syncable patients for a region. They are both conceptually sound. Although, I seem to have a slight preference of how we had it before, purely because following the codepaths overall, there's less indirection.

Don't think there will be multiple queries no?

I think, `fg.syncable_patients` should present the same time-complexity of a query like `Patient.where(reg_facility: fg.facilities)`.

I have another idea, that I didn't explore to keep it simple, that I'll take a shot at and see if y'all like it.